<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>450</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>NIELIT-Calicut, Cyber Physical System Lab</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Robot GUI Development/images.jpeg</normaloff>../../Robot GUI Development/images.jpeg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>451</width>
      <height>291</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier 10 Pitch</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="tabletTracking">
     <bool>true</bool>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="cursorWidth">
     <number>7</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Single Step Run</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>NT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>110</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>GC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>200</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>GO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>340</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Run as Single Program </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>310</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>ABORT PROGRAM</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuSave_Options">
     <property name="title">
      <string>Save Options</string>
     </property>
     <addaction name="actionSave_Program_2"/>
     <addaction name="actionSave_As"/>
     <addaction name="separator"/>
    </widget>
    <addaction name="actionNew_Program"/>
    <addaction name="actionOpen_Program"/>
    <addaction name="separator"/>
    <addaction name="menuSave_Options"/>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuAbout_2">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_this_Software"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
   <addaction name="menuAbout_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_Program">
   <property name="icon">
    <iconset>
     <normaloff>../3rd party/Qt-91-e1e826bfb283ced96d7772941c0a8acbdd3fcad7/Qt 91/new.png</normaloff>../3rd party/Qt-91-e1e826bfb283ced96d7772941c0a8acbdd3fcad7/Qt 91/new.png</iconset>
   </property>
   <property name="text">
    <string>New Program</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen_Program">
   <property name="icon">
    <iconset>
     <normaloff>../3rd party/Qt-91-e1e826bfb283ced96d7772941c0a8acbdd3fcad7/Qt 91/open.png</normaloff>../3rd party/Qt-91-e1e826bfb283ced96d7772941c0a8acbdd3fcad7/Qt 91/open.png</iconset>
   </property>
   <property name="text">
    <string>Open Program</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="ResourcesImg.qrc">
     <normaloff>:/imgresrc/Icons/Icons/DeleteSticky.bmp</normaloff>:/imgresrc/Icons/Icons/DeleteSticky.bmp</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset>
     <normaloff>../3rd party/Qt-91-e1e826bfb283ced96d7772941c0a8acbdd3fcad7/Qt 91/cut.png</normaloff>../3rd party/Qt-91-e1e826bfb283ced96d7772941c0a8acbdd3fcad7/Qt 91/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset>
     <normaloff>../3rd party/Qt-91-e1e826bfb283ced96d7772941c0a8acbdd3fcad7/Qt 91/copy.png</normaloff>../3rd party/Qt-91-e1e826bfb283ced96d7772941c0a8acbdd3fcad7/Qt 91/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset>
     <normaloff>../3rd party/Qt-91-e1e826bfb283ced96d7772941c0a8acbdd3fcad7/Qt 91/paste.png</normaloff>../3rd party/Qt-91-e1e826bfb283ced96d7772941c0a8acbdd3fcad7/Qt 91/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset>
     <normaloff>../3rd party/Qt-91-e1e826bfb283ced96d7772941c0a8acbdd3fcad7/Qt 91/redo.png</normaloff>../3rd party/Qt-91-e1e826bfb283ced96d7772941c0a8acbdd3fcad7/Qt 91/redo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset>
     <normaloff>../3rd party/Qt-91-e1e826bfb283ced96d7772941c0a8acbdd3fcad7/Qt 91/undo.png</normaloff>../3rd party/Qt-91-e1e826bfb283ced96d7772941c0a8acbdd3fcad7/Qt 91/undo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionSave_Program_2">
   <property name="icon">
    <iconset>
     <normaloff>../3rd party/Qt-91-e1e826bfb283ced96d7772941c0a8acbdd3fcad7/Qt 91/save.png</normaloff>../3rd party/Qt-91-e1e826bfb283ced96d7772941c0a8acbdd3fcad7/Qt 91/save.png</iconset>
   </property>
   <property name="text">
    <string>Save Program</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset>
     <normaloff>../3rd party/Qt-91-e1e826bfb283ced96d7772941c0a8acbdd3fcad7/Qt 91/saveas.png</normaloff>../3rd party/Qt-91-e1e826bfb283ced96d7772941c0a8acbdd3fcad7/Qt 91/saveas.png</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionAbout_this_Software">
   <property name="icon">
    <iconset>
     <normaloff>../3rd party/Qt-91-e1e826bfb283ced96d7772941c0a8acbdd3fcad7/Qt 91/about.png</normaloff>../3rd party/Qt-91-e1e826bfb283ced96d7772941c0a8acbdd3fcad7/Qt 91/about.png</iconset>
   </property>
   <property name="text">
    <string>About this Software</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+I</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="ResourcesImg.qrc"/>
 </resources>
 <connections/>
</ui>
